app/util/smtp.py

```diff
@@ -0,0 +1,39 @@
+import smtplib
+
+
+class SMTP():
+    def __init__(self, host, port, user, password):
+        self.conn = None
+        self.host = host
+        self.port = port
+        self.user = user
+        self.password = password
+
+    def open(self):
+        if self.conn:
+            return False
```

Comment:
This is weird, does the function return boolean type or connection? Also what if the connection was reset at that point? The object would still be there but any attempt to send a message through it would fail.

Reply:
fixed